docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line, and also
+# from the environment for the first two.
+SPHINXOPTS    ?=
+SPHINXBUILD   ?= sphinx-build
+SOURCEDIR     = .
+BUILDDIR      = build
```

Comment:
I don't know what is standard procedure here but looks a bit ugly with the spacing - maybe line up the `=`

Reply:
That `Makefile` is created by Sphinx automatically with the default configuration.